pkg/vm/interop.go

```diff
@@ -50,3 +87,29 @@ func RuntimeDeserialize(vm *VM) error {
 
 	return nil
 }
+
+// init initializes the global defaultVMInterops value.
+func init() {
+	defaultVMInterops = make([]interopIDFuncPrice, 0, 6) // Registering 6 interops down below.
+	defaultVMInterops = append(defaultVMInterops,
```

Comment:
I think it's nice to have Map like []byte("Neo.Runtime.Log") : InteropFuncPrice{runtimeLog, 1}}
and in `defaultVMInterops = make([]interopIDFuncPrice, 0, 6)` use size of that map and have foreach loop with appends. WDYT?

Reply:
Yeah, this one doesn't look good, but I think I have a bit better solution, will update.